### ISSUE TITLE
Tests fail with Moo 2.002002

### ISSUE THREAD

**Issue by eserte** [pdl/Test-Proto#44]
My smoker machines started to report the following test failure:

```
Class::XSAccessor: invalid instance method invocant: no hash ref supplied at /home/cpansand/.cpan/build/2016062218/Test-Proto-0.02
7-rhfIDA/blib/lib/Test/Proto/Base.pm line 203.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 255 just after 1.
t/Test-Proto-ArrayRef.t ............... 
Dubious, test returned 255 (wstat 65280, 0xff00)
All 1 subtests passed 
... (etc., all subsequent tests fail in the same way) ...
```

It seems that the problem is caused by the latest Moo version. Statistical analysis of test reports from my machine (negative theta is bad):

```
****************************************************************
Regression 'mod:Moo'
****************************************************************
Name                   Theta          StdErr     T-stat
[0='const']           1.0000          0.0000    12920882287520642.00
[1='eq_1.004000']             0.0000          0.0000       0.81
[2='eq_1.004001']             0.0000          0.0000       0.81
[3='eq_1.004002']            -0.0000          0.0000      -0.45
[4='eq_1.004005']             0.0000          0.0000       1.08
[5='eq_1.006000']            -0.0000          0.0000      -0.48
[6='eq_1.006001']             0.0000          0.0000       0.00
[7='eq_1.007000']             0.0000          0.0000       2.15
[8='eq_2.000001']             0.0000          0.0000       1.08
[9='eq_2.000002']             0.0000          0.0000       2.84
[10='eq_2.001001']            0.0000          0.0000       5.21
[11='eq_2.002002']           -1.0000          0.0000    -11506866883889730.00

R^2= 1.000, N= 86, K= 12
****************************************************************
```

**Comment by haarg** [comment]
Fixed in Moo 2.002003.

**Comment by pdl** [comment]
@eserte and @haarg - belated thanks.
